### PR TITLE
feat: add page for create a task/job title on client side [sc-269]

### PR DESCRIPTION
**Pull request by marialobillo**
Screenshot 2024-05-17 at 17 44 07

### CODE REVIEW

File: client/src/App.tsx [hunk 1]
```diff
@@ -1,33 +1,17 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
 import './App.css'
+import { Routes, Route } from 'react-router-dom'
+import CreateJob from './pages/CreateJob'
```
Comment (AbdelrahmanRadwan) [inline client/src/App.tsx]: I think we want to call it a task instead of a job?

File: client/src/pages/CreateJob.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+
+
+const CreateJob = () => {
```
Comment (AbdelrahmanRadwan) [inline client/src/pages/CreateJob.tsx]: Can you also add the css file for this?

Comment (AbdelrahmanRadwan) [reply client/src/pages/CreateJob.tsx]: In the CSS, add different colour to the button, padding, and margin.

File: client/src/pages/CreateJob.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  const [jobTitle, setJobTitle] = useState<string>('');
+
+  const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setJobTitle(event.target.value)
+  }
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault()
+    console.log('Job Title:', jobTitle )
```
Comment (AbdelrahmanRadwan) [inline client/src/pages/CreateJob.tsx]: The console log should be removed after you have tested.

File: client/src/pages/CreateJob.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+      <form action="">
+        <input 
+          type="text"
+          value={jobTitle}
+          onChange={handleInputChange} 
+          placeholder="Enter job title" />
+        <button type="submit">Create Job</button>
+      </form>
+      
+      <p>Job Title: {jobTitle}</p>
```
Comment (AbdelrahmanRadwan) [inline client/src/pages/CreateJob.tsx]: A task should be defined by multiple things, not just the title.
- Title
- Description
- Location (can be just text)
- Price
- By when should it be done.
- Images and media that the user would upload.

Some of the fields can be optinal.

File: client/src/App.tsx (hunk 1 above)
Comment (marialobillo) [reply client/src/App.tsx]: Yes, task is better, it's done

File: client/src/pages/CreateJob.tsx (hunk 3 above)
Comment (marialobillo) [reply client/src/pages/CreateJob.tsx]: yes, removed it.

File: client/src/pages/CreateJob.tsx (hunk 2 above)
Comment (marialobillo) [reply client/src/pages/CreateJob.tsx]: I have added tailwind, or what do you mean is for every page/component add a css file?

Comment (AbdelrahmanRadwan) [reply client/src/pages/CreateJob.tsx]: I meant to add CreateJob.css or something like that.

File: client/src/pages/CreateTask.tsx [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+        <div className="grid md:grid-cols-2 md:gap-6">
+        <div>
+          <label 
+            htmlFor="title"
+            className="block mb-2 text-sm font-medium text-gray-900 dark:text-white"
+            >Title</label>
+          <input
+            type="text"
+            name="title"
+            className="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"
```
Comment (AbdelrahmanRadwan) [inline client/src/pages/CreateTask.tsx]: It's a bad practice to add the css stying here, pleae separate that to a css file and use newer css features.

Comment (marialobillo) [reply client/src/pages/CreateTask.tsx]: Ok, thank you, I will

File: client/src/pages/CreateJob.tsx (hunk 4 above)
Comment (marialobillo) [reply client/src/pages/CreateJob.tsx]: I think this one is done as well. All the attributes are in the form.

File: client/src/pages/CreateTask.tsx (hunk 5 above)
Comment (marialobillo) [reply client/src/pages/CreateTask.tsx]: I added a css file for the createTask page, thank you for the css :)

File: client/src/pages/CreateJob.tsx (hunk 2 above)
Comment (marialobillo) [reply client/src/pages/CreateJob.tsx]: This one is answered later

**Review by AbdelrahmanRadwan — APPROVED** [review]

**Pull request merged**